contrib/opencensus-ext-google-cloud-clientlibs/setup.py

```diff
@@ -39,9 +39,9 @@
     include_package_data=True,
     long_description=open('README.rst').read(),
     install_requires=[
-        'opencensus >= 0.4.dev0, < 1.0.0',
-        'opencensus-ext-grpc >= 0.2.dev0, < 1.0.0',
-        'opencensus-ext-requests >= 0.2.dev0, < 1.0.0',
+        'opencensus >= 0.3.0, < 1.0.0',
```

Comment:
For ext packages that don't depend on changes in the core library I didn't change the required opencensus version. This could get confusing quickly since this requirement will be `0.5.dev0` on master to allow us to keep making breaking changes on the dev version.

We may want to keep these version numbers in sync across all the ext packages in the interest of our sanity, at the cost of backwards compatibility.